arch/risc-v/src/common/riscv_exception_common.S

```diff
@@ -36,7 +36,15 @@
 #  define REGLOAD   ld
 #  define REGSTORE  sd
 #endif
- 
+
+#ifdef CONFIG_N_IRQ_STACKS
```

Comment:
CONFIG_N_IRQ_STACKS->CONFIG_IRQ_NSTACKS and N_IRQ_STACKS->IRQ_NSTACKS?

Reply:
Sure!